db/config.cc

```diff
@@ -1456,6 +1456,7 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , disk_space_monitor_normal_polling_interval_in_seconds(this, "disk_space_monitor_normal_polling_interval_in_seconds", value_status::Used, 10, "Disk-space polling interval while below polling threshold")
     , disk_space_monitor_high_polling_interval_in_seconds(this, "disk_space_monitor_high_polling_interval_in_seconds", value_status::Used, 1, "Disk-space polling interval at or above polling threshold")
     , disk_space_monitor_polling_interval_threshold(this, "disk_space_monitor_polling_interval_threshold", value_status::Used, 0.9, "Disk-space polling threshold. Polling interval is increased when disk utilization is greater than or equal to this threshold")
+    , critical_disk_utilization_level(this, "critical_disk_utilization_level", liveness::LiveUpdate, value_status::Used, 0.98, "Disk utilization level above which all mutation writes to user table are rejected to prevent out-of-space errors")
```

Comment:
Let's have a more general description and point to the documentation for more details about what the critical level controls.

Reply:
Let's consider either a .md document (in docs/dev) or a user visible .rst doc for user-oriented documentation.

Cc @annastuchlik